### PR TITLE
Chore (profile): Add profile for Ruairidh Orr Ewing (#645)

### LINKED ISSUE #645: Data Scientist position - Ruairidh Orr Ewing
### First Name

Ruairidh

### Last Name

Orr Ewing

### Telegram

Ruairidh_OE

### How did you find out about this job?

Linkedin 

### Link to Portfolio

https://github.com/roars2121

### Link to LinkedIn Profile

https://www.linkedin.com/in/ruairidh-orr-ewing-4332851b5

### Short Description

I am currently pursuing a part-time Master's degree in biomedical engineering, with coursework in artificial intelligence and data analytics.  I also hold a Data Scientist Associate certificate from DataCamp. My GitHub portfolio showcases several developed models, including a random forest regressor for house price prediction (RMSE: $15,000) and a supervised learning model for facial recognition (82% accuracy achieved through hyperparameter tuning). My qualifications are well-suited to this role; I am a high-achieving, quick learner with a proven track record of exceeding expectations (200% target attainment in sales). My technical skills encompass Python (scikit-learn, TensorFlow), R (supervised and unsupervised learning models for medical image analysis, including malignant tumor prediction), and SQL.

[Updated ROE CV.pdf](https://github.com/user-attachments/files/20829282/Updated.ROE.CV.pdf)

### Application Position

Data Scientist

### PR DESCRIPTION
**Pull request by roars2121**
Added Profile to profile-submission.json
closes #645 

## Summary by CodeRabbit

- **Chores**
  - Added a new team profile for Ruairidh Orr Ewing to the team profiles list.

### CODE REVIEW

**Comment by roars2121** [comment]
@coderabbitai review

**Review by tiffmyaaw — CHANGES_REQUESTED** [review]
Apply the `coderabbitai` bot suggestion.

Rename your PR title and description based on our [Guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md#naming)

Also edit the PR description to [link your issue manually](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue).

Last, submit your time spent on the PR Time Tracker bot.

Feel free to look at the successfully closed PRs in this repo for refs.

File: profile-submission.json [hunk 1]
```diff
@@ -234,6 +234,10 @@
       "github_handle": "Harry-027",
       "full_name": "Harish Bhawnani",
       "github_trial_issue_link": "https://github.com/holdex/trial/issues/624"
+    },{
```
Comment (tiffmyaaw) [inline profile-submission.json:237]: the { should be in a new line

**Review by tiffmyaaw — CHANGES_REQUESTED** [review]
